src/dmd/dmodule.d

```diff
@@ -624,7 +624,9 @@ extern (C++) final class Module : Package
             {
                 .error(loc, "cannot find source code for runtime library file 'object.d'");
                 errorSupplemental(loc, "dmd might not be correctly installed. Run 'dmd -man' for installation instructions.");
-                errorSupplemental(loc, "config file: %s", FileName.canonicalName(global.inifilename));
+                const dmdConfFile = FileName.canonicalName(global.inifilename);
+                if (dmdConfFile)
+                    errorSupplemental(loc, "config file: %s", dmdConfFile);
```

Comment:
Why not keep the existing behaviour with sth. like: `dmdConfFile ? dmdConfFile : "(null)".ptr`?

Reply:
I'm not fussed either way, just so long as it isn't null.